### PR TITLE
feature(FDS-342): Add input month field

### PR DESCRIPTION
**Pull request by rociobaena**
*** Merge after https://github.com/Espressive/cascara/pull/261/files -- Input image module

The input month type is adding to our module list.
This PR includes:
- Module DataMonth.
- Fixture
- Unit test

### Snapshots
A new fixture for the DataMonth field is created.

## New Component Checklist

- [x] Includes unit tests for _ALL_ required FDS use cases
- [x] Does not mute any top level API props in React
- [x] Includes an explanation for any changes to Jest snapshots (should be a PR tag automatically added if this happens)
- [-] Component uses `React.forwardRef()` when a single DOM node is returned, _AND_ it makes React-sense to access that DOM node

### CODE REVIEW

File: packages/cascara/src/modules/DataImage/DataImage.fixture.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import DataImage from './DataImage';
+import ModuleSandbox from '../ModuleSandbox';
+
+const DisplayDataImage = ({ isEditing, value, ...rest }) => (
```
Comment (lordpixel) [inline packages/cascara/src/modules/DataImage/DataImage.fixture.js:5]: @rociobaena should we keep `value` in `...rest`?

File: packages/cascara/src/modules/DataImage/DataImage.fixture.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  <ModuleSandbox isEditing={isEditing}>
+    <DataImage {...rest} label='My button' />
+  </ModuleSandbox>
+);
+
+const DataDateModule = () => {
+  return (
+    <>
+      <div>
+        <h1>Date input field</h1>
```
Comment (lordpixel) [inline packages/cascara/src/modules/DataImage/DataImage.fixture.js:15]: I think this should be `Image input field`

```suggestion
        <h1>Image input field</h1>
```

File: packages/cascara/src/modules/DataImage/DataImage.fixture.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <p>
+          This fixture is for testing the presentation of an input date field
```
Comment (lordpixel) [inline packages/cascara/src/modules/DataImage/DataImage.fixture.js:17]:
```suggestion
          This fixture is for testing the presentation of an image input field
```

**Review by lordpixel — DISMISSED** [review]
@rociobaena I left some change suggestions 👍

File: packages/cascara/src/modules/DataImage/DataImage.fixture.js (hunk 1 above)
Comment (brianespinosa) [reply packages/cascara/src/modules/DataImage/DataImage.fixture.js:5]: Definitely more simple to keep everything in rest except for the value that needs to come out to go to the provider

**Review by brianespinosa — DISMISSED** [review]

File: packages/cascara/src/modules/DataImage/DataImage.fixture.js (hunk 2 above)
Comment (rociobaena) [reply packages/cascara/src/modules/DataImage/DataImage.fixture.js]: New fixture file

**Review by brianespinosa — DISMISSED** [review]

**Review by brianespinosa — APPROVED** [review]

**Review by brianespinosa — APPROVED** [review]
@lordpixel looks like this one needs another approval from you.

**Review by brianespinosa — APPROVED** [review]

**Review by lordpixel — APPROVED** [review]

**Pull request merged**